vowel_count_code.py

```diff
@@ -0,0 +1,4 @@
+import re
+
+def vowel_count(string):
+    return len(re.findall(r'[aeiouAEIOU]', string))
```

Comment:
This is a fine solution, I'd like to see you do it by checking for inclusion within a set that holds the vowels and a for loop over the string incrementing a variable holding an integer as a counter any time the character matches.

Once you have done the above, and since you are also learning `re`, count all the vowels in a string but only if they are followed immediately by the letter `x`.  You don't have to commit this to github, just send it to me in slack.

Also given this this r-string '^[a-z][a-z_0-9]+$' what are some strings that will match and what are some that will not?